docs/docusaurus/docs/contributing/application_integration_support.md

```diff
@@ -0,0 +1,100 @@
+
+| GX supported                             | Community supported                                                |
+|------------------------------------------|--------------------------------------------------------------------|
+| <i class="fas fa-check-circle" style={{color: "#28a745"}}></i> macOS<br/><i class="fas fa-check-circle" style={{color: "#28a745"}}></i> Linux    | <i class="fas fa-circle" style={{color: "#dc3545"}}></i> Windows              | 
+
+## Python versions
+
+The following table defines the Python versions supported by GX and the community. GX typically follows the [Python release cycle](https://devguide.python.org/versions/).
+
+| GX supported                             | Community supported                                                |
+|------------------------------------------|--------------------------------------------------------------------|
+| <i class="fas fa-check-circle" style={{color: "#28a745"}}></i> 3.8 to 3.11    | <i class="fas fa-circle" style={{color: "#dc3545"}}></i> 3.7.X and earlier              | 
+
+## Datasources
+
+The following table defines the Datasources supported by GX and the community.
+
+| GX supported                             | Community supported                                                |
+|------------------------------------------|--------------------------------------------------------------------|
+| <i class="fas fa-circle" style={{color: "#dc3545"}}></i> Snowflake<br/><i class="fas fa-circle" style={{color: "#dc3545"}}></i> Databricks<br/><i class="fas fa-circle" style={{color: "#dc3545"}}></i> BigQuery<br/><i class="fas fa-check-circle" style={{color: "#28a745"}}></i> PostgreSQL<br/><i class="fas fa-check-circle" style={{color: "#28a745"}}></i> pandas (maintenance mode)<br/><i class="fas fa-check-circle" style={{color: "#28a745"}}></i> Spark (maintenance mode)  | <i class="fas fa-check-circle" style={{color: "#28a745"}}></i> Redshift<br/><i class="fas fa-check-circle" style={{color: "#28a745"}}></i> MSSQL<br/><i class="fas fa-check-circle" style={{color: "#28a745"}}></i> MySQL<br/><i class="fas fa-check-circle" style={{color: "#28a745"}}></i> SQLite<br/><i class="fas fa-check-circle" style={{color: "#28a745"}}></i> Trino<br/><i class="fas fa-check-circle" style={{color: "#28a745"}}></i> AWS S3<br/><i class="fas fa-check-circle" style={{color: "#28a745"}}></i> Google Cloud Storage<br/><i class="fas fa-check-circle" style={{color: "#28a745"}}></i> Azure Blog Storage<br/><i class="fas fa-circle" style={{color: "#ffc107"}}></i> Clickhouse<br/><i class="fas fa-circle" style={{color: "#ffc107"}}></i> Athena<br/><i class="fas fa-circle" style={{color: "#ffc107"}}></i> Dremio<br/><i class="fas fa-circle" style={{color: "#ffc107"}}></i> Teradata<br/><i class="fas fa-circle" style={{color: "#ffc107"}}></i> Vertica<br/><i class="fas fa-circle" style={{color: "#dc3545"}}></i> EMR Spark<br/><i class="fas fa-circle" style={{color: "#dc3545"}}></i> AWS Glue             | 
```

Comment:
Should Databricks, Snowflake, and Bigquery be yellow here? Snowflake and BigQuery are certainly under partial test, and we support all three of them.